contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_buildozer.py

```diff
@@ -65,71 +64,37 @@ def test_execute_binary(self):
-    roots = ['e']
-    dependency_to_add = '/o/p/q'
-
-    self._test_add_dependencies_with_targets([dependency_to_add], roots, targets)
+    targets = ['b', 'c']
 
-  def test_implicit_name_among_rules(self):
-    self.add_to_build_file('f', 'java_library(name="f")')
-    self.add_to_build_file('g', 'java_library(name="g")')
-    self.add_to_build_file('h', 'java_library()')
+    dependency_to_add = '/l/m/n'
 
-    targets = { 'e': self.make_target('e'), 'g': self.make_target('g'), 'h': self.make_target('h') }
-    roots = ['h']
-    dependency_to_add = '/r/s/t'
+    self._run_buildozer({ 'add_dependencies': dependency_to_add }, targets=targets)
 
-    self._test_add_dependencies_with_targets([dependency_to_add], roots, targets)
+    for target in targets:
+      self.assertInFile(dependency_to_add, '{}/{}/BUILD'.format(self.build_root, target))
 
   def _test_add_dependencies(self, spec, dependencies_to_add):
     self._run_buildozer({ 'add_dependencies': ' '.join(dependencies_to_add) })
 
     for dependency in dependencies_to_add:
-      self.assertIn(dependency, self._build_file_dependencies(os.path.join(self.build_root, spec, 'BUILD')))
-
-  def _test_add_dependencies_with_targets(self, dependencies_to_add, roots, targets):
-    """
-    Test that a dependency is (or dependencies are) added to a BUILD file with buildozer.
-    This can run on multiple context roots and multiple target objects.
-    """
-    for dependency_to_add in dependencies_to_add:
-      self._run_buildozer({ 'add_dependencies': dependency_to_add }, roots=roots, targets=targets)
-
-    for root in roots:
-      self.assertInFile(dependency_to_add, os.path.join(self.build_root, root, 'BUILD'))
+      self.assertIn(dependency, self._build_file_dependencies('{}/{}/BUILD'.format(self.build_root, spec)))
 
   def _test_remove_dependencies(self, spec, dependencies_to_remove):
-    self._run_buildozer({ 'remove_dependencies': ' '.join(dependencies_to_remove) }, roots=[spec])
+    self._run_buildozer({ 'remove_dependencies': ' '.join(dependencies_to_remove) }, targets=[spec])
 
     for dependency in dependencies_to_remove:
-      self.assertNotIn(dependency, self._build_file_dependencies(os.path.join(self.build_root, spec, 'BUILD')))
-
-  def _run_buildozer(self, options, roots=['b'], targets=None):
-    """Run buildozer on the specified context roots and target objects.
-
-    roots -- the context roots supplied to buildozer (default ['b'])
-    targets -- the targets buildozer will run on (defaults to self.targets)
-    """
-    targets = self.targets if targets is None else targets
+      self.assertNotIn(dependency, self._build_file_dependencies('{}/{}/BUILD'.format(self.build_root, spec)))
 
+  def _run_buildozer(self, options, targets=['b']):
     self.set_options(**options)
 
     target_roots = []
 
-    for root in roots:
-      target_roots.append(targets[root])
+    for root in targets:
+      target_roots.append(self.targets[root])
```

Comment:
The refactoring here makes it more confusing, also have `targets` default to `['b']` does not make sense. would recommend reverting then modifying the caller.